src/main/scala/chisel3/testers/TesterDriver.scala

```diff
@@ -13,6 +13,7 @@ import firrtl.options.{Dependency, Phase, PhaseManager, TargetDirAnnotation, Uns
 import firrtl.stage.{FirrtlCircuitAnnotation, FirrtlStage}
 import firrtl.transforms.BlackBoxSourceHelper.writeResourceToDirectory
 
+@deprecated("This is not a User API, will be removed in Chisel 3.6", "3.5")
```

Comment:
Let's do these deprecations separately. These aren't great APIs and they're not commonly used but they _are_ used and thus we need a migration path if we're going to plan to remove them.

So let's not do the deprecation here, but get this PR merged.

You could just copy these things into `src/test/scala` and modify there, but alternatively you could factor out the functionality into a package private trait that `TesterDriver` extends and then do the same thing in `src/test/scala/chisel3` and provide the modified functionality that way.